insights/parsers/crypto_policies.py

```diff
@@ -68,27 +68,55 @@ def parse_content(self, content):
 
 
 @parser(Specs.crypto_policies_opensshserver)
-class CryptoPoliciesOpensshserver(SysconfigOptions):
+class CryptoPoliciesOpensshserver(Parser):
     """
     This parser reads the ``/etc/crypto-policies/back-ends/opensshserver.config``
     file.  It uses the ``SysconfigOptions`` parser class to convert the file into
     a dictionary of options. It also provides the ``options`` property as a helper
     to retrieve the ``CRYPTO_POLICY`` variable.
 
-    Sample Input::
+    Sample Input on RHEL8::
 
         CRYPTO_POLICY='-oCiphers=aes256-gcm@openssh.com,3des-cbc -oMACs=umac-128-etm@openssh.com'
 
+    Sample Input on RHEL9::
+
+        Ciphers aes256-gcm@openssh.com,chacha20-poly1305@openssh.com,aes256-ctr,aes128-gcm@openssh.com,aes128-ctr
+        MACs hmac-sha2-256-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha2-256,hmac-sha1,umac-128@openssh.com,hmac-sha2-512
+
     Examples:
-        >>> 'CRYPTO_POLICY' in cp_os
+        >>> 'CRYPTO_POLICY' in cp_os.data
```

Comment:
Since `data` cannot indicate the type of data stored, it is no longer a suggested name for a Parser attribute.  In this case, how about making the parser as `dict` directly, e.g `class CryptoPoliciesOpensshserver(Parser, dict):`.

Reply:
@xiangce - Updated, please check